docs/cicd/cicd-spans.md

```diff
@@ -0,0 +1,106 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Spans
+--->
+
+# Semantic conventions for CICD spans
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [CICD Spans](#cicd-spans)
+  - [Pipeline run](#pipeline-run)
+  - [Pipeline task run](#pipeline-task-run)
+
+<!-- tocstop -->
+
+## CICD Spans
+
+The conventions described in this section are specific to Continuous Integration / Continuous Deployment (CICD) systems.
+
+**Disclaimer:** These are initial CICD spans and attributes
+but more may be added in the future.
+
+Also consider setting other attributes of the [CICD and VCS resource conventions][cicdres].
+
+[cicdres]: /docs/resource/cicd.md (CICD and VCS resource conventions)
+
+### Pipeline run
+
+<!-- semconv span.cicd.pipeline.run.server -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`cicd.pipeline.result`](/docs/attributes-registry/cicd.md) | string | The result of a pipeline run. | `success`; `failure`; `timeout`; `skipped` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
I do not see `cancelation` and `error` described below

Reply:
It's described a few lines below: https://github.com/open-telemetry/semantic-conventions/blob/bbce19df8fecc23c220c58ca8f73f9c80a73361a/docs/cicd/cicd-spans.md?plain=1#L43-L52

Do you mean that the description should be improved?

Reply:
For some reason, I thought it was a complete list of values instead of a sample

Reply:
Is it good as it is or should I improve the description?